src/mmw/apps/geoprocessing_api/tasks.py

```diff
@@ -278,15 +278,18 @@ def analyze_terrain(result):
 
     [elevation, slope] = result
 
+    def cm_to_m(x):
+        x * M_PER_CM if x else None
```

Comment:
Missing `return` is making all the elevation values 0.

Reply:
Haha, that's embarrassing. Functional summer has done its work.